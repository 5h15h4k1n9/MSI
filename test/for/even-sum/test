sumEven(6)
exit
