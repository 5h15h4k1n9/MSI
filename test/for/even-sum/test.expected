(int) 12
Bye-bye!
